includes/commerce_pos.transaction.inc

```diff
@@ -936,3 +937,80 @@ function _commerce_pos_override_confirm_validate($form, &$form_state) {
   // Add the product to the overridden items list.
   $form_state['out_of_stock_overridden_products'][$form_state['triggering_element']['#product_id']] = $form_state['triggering_element']['#product_id'];
 }
+
+/**
+ * Returns a modal form displaying each item in an order along with a checkbox
```

Comment:
The docblock should have a one-line summary under 80 characters long. Then a longer description underneath. Also, it would be useful to include the $order as a @param